Tests/CodeGen_MonoGameForms_FullCodegen/Content/Components/Spaced Component.gucx

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ComponentSave xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <Name>Spaced Component</Name>
+  <BaseType>Container</BaseType>
+  <State>
+    <Name>Default</Name>
+    <Variable>
+      <Type>Spaced Category</Type>
+      <Name>Spaced CategoryState</Name>
```

Comment:
Gum will probably remove this space on the next startup of the project, so that may undo this. It's probably best to remove this variable.